.appveyor.yml

```diff
@@ -52,19 +50,13 @@ install:
   - conda config --prepend channels conda-forge
 
   # For building, use a new environment
-  - conda create -q -n test-environment python=%PYTHON_VERSION% tk "pip<22.0"
-  - activate test-environment
+  - echo ^ ^ - python=%PYTHON_VERSION% >> environment.yml
```

Comment:
I guess the `^` is an escape, but can you not quote the line instead?

Reply:
I tried... the quotes show up in the output

Reply:
This is slightly hackish, but couldn't figure out a better way to specify python version while also using environment.yml

If you don't specify it will get the latest version and then need to reinstall many things to get versions for older python.